tests/core/fs/test_fs_online_rwfile_and_sync.py

```diff
@@ -13,6 +15,47 @@
 PLAYGROUND_SIZE = BLOCK_SIZE * 10
 
 
+@pytest.mark.trio
+@pytest.mark.slow
```

Comment:
Don't you need `@customize_fixture(real_data_storage=True)` ? Otherwise the sqlite is in memory ? (or this is no longer needed because the new rust storage doesn't use this customization ? but in that case do we need to do something as documentation/deprecation/change in another PR ?)

Reply:
🤷 anyway this test is detecting file descriptor leaks, even with the fixture customization it currently leaks fd